### PR TITLE
Profiles counter on home page

### PR DESCRIPTION
**Pull request by henrique-borba**
## Profiles counter on home page

This PR adds the profile counter on the main page, a shortcut for managing profiles with the main objective of reminding the user that he does not have registered profiles yet.

Screenshot_20210824_143620

## Solves

- https://github.com/jasonmunro/cypht/issues/380

### CODE REVIEW

File: modules/nux/modules.php [hunk 1]
```diff
@@ -71,12 +72,20 @@ public function process() {
         if (data_source_available($modules, 'smtp')) {
             $smtp_servers = count(Hm_SMTP_List::dump(false));
         }
```
Comment (jasonmunro) [inline modules/nux/modules.php:74]: Looks like the smtp section got duplicated here and we should set a default empty value for $profiles if the site does not have that module set enabled.

**Comment by henrique-borba** [comment]
@jasonmunro there you go

**Comment by jasonmunro** [comment]
> @jasonmunro there you go

excellent thanks for doing this!

**Comment by marclaporte** [comment]
This is a very very good idea!

**Pull request merged**